File: Website/src/app/models/lesson-plan.output-model.ts
import { ClassOutputModel } from "./class.output-model";
import { SubjectOutputModel } from "./subject.output-model";
import { TeacherOutputModel } from "./teacher.output-model";

export class LessonPlanOutputModel {
    teacher: TeacherOutputModel;
    subject: SubjectOutputModel;
    class: ClassOutputModel;
    dayOfWeek: number;
    lessonStart: Date;
    lessonStop: Date;
}

File: ChildManager.Backend/Models/LessonPlanOutputModel.cs
using System;

namespace ChildManager.Models
{
    public class LessonPlanOutputModel
    {
        public TeacherOutputModel Teacher { get; set; }
        public SubjectOutputModel Subject { get; set; }
        public ClassOutputModel Class { get; set; }
        public DayOfWeek DayOfWeek { get; set; }
        public DateTime LessonStart { get; set; }
        public DateTime LessonStop { get; set; }
    }
}
